www/cap.html.php

```diff
@@ -38,18 +38,18 @@
 	echo '<div style="text-align: right;">', "\n";
 	echo '<big><big><span style="font-weight: bold;">'._("Identity Verification Form (CAP) form").'</span></big></big><br>', "\n";
 	echo '</div>', "\n";
-	echo '<div style="text-align: right;">'.'Hangar 10 Airfield Avenue, Murwillumbah NSW 2484, New South Wales, (Commonwealth of) Australia  - <a href="http://www.cacert.org/"> http://www.cacert.org/</a><br></div>', "\n";
+	echo '<div style="text-align: right;">'.'Clos Belmont 2, 1208 Geneva, Switzerland  - <a href="http://www.cacert.org/"> http://www.cacert.org/</a><br></div>', "\n";
```

Comment:
Switzerland or Suisse?

Reply:
if it is written in Englisch: Geneva, Switzerland.